app/src/components/Project/Header.tsx

```diff
@@ -3,34 +3,46 @@ import Logo from '../ui/Logo'
 import UserPanel from '../User/UserPanel'
 import { useAuth } from '../../context/AuthContext'
 import { useProject } from '../../context/ProjectContext'
+import { SearchIcon, RingIcon } from '../icons'
 
 export function ProjectHeader() {
   const { id: projectId = '' } = useParams()
   const navigate = useNavigate()
   const { user } = useAuth()
   const { search, setSearch } = useProject()
+  const badgeCount = 5

```

Comment:
constants could be outside the component

Reply:
stop why is it a constant?

Reply:
This is temporary, I just don’t quite understand what we’re calculating here.

Reply:
Actually that's [notifications](https://www.figma.com/design/EJ1Z3dS3nGHxdwbHaNgN1K/dun.wtf?node-id=5941-46519&t=SdpwG0Q2ojabh8pJ-4). But let's discuss them later. Now we could hide this button, because it doesn't make sense 🥸